wurf_cxx_mkspec.py

```diff
@@ -108,6 +108,11 @@ def options(opt):
         '--emscripten_path', default=None, dest='emscripten_path',
         help="Path to the Emscripten compiler (em++)")
 
+    opts.add_option(
+        '--poky_sdk_path', default=None, dest='poky_sdk_path',
+        help="Configure toolchain (cross compiler etc.) for the Yocto based "
```

Comment:
```suggestion
        help="Path to the Yocto-based cross-compiler toolchain for the  "
```
I tried to harmonize the description with the other sdk path options above.